spyderlib/app/cli_options.py

```diff
@@ -37,5 +37,7 @@ def get_options():
     parser.add_option('--profile', action='store_true', default=False,
                       help="Profile mode (internal test, "
                            "not related with Python profiling)")
+    parser.add_option('--windowtitle', type=str, default=None,
```

Comment:
Could add an hyphen between `window` and `title`? So this option should be

    --window-title